configs/setup_golden_image.py

```diff
@@ -138,7 +143,11 @@ def debootstrap_image(manconfig, vmd_path=None):
         return True
 
     errors = [ e for e in contents if 'WARN' in e or 'ERROR' in e ]
-    #logging.error(''.join(errors), file=sys.stderr)  # already have newlines
+    LOGGER = logging.tmmsLogger('golden', destdir + '/build.log')
+
+    if LOGGER is not None: #sanity
+        LOGGER.error(''.join(errors), file=sys.stderr)  # already have newlines
+
     raise RuntimeError('vmdebootstrap failed, consult %s' % vmdlog)
```

Comment:
"errors" list is not being used?

Reply:
No, cause it is saved in the vmdlog file.